libsolidity/ast/ASTJsonExporter.cpp

```diff
@@ -286,6 +286,8 @@ bool ASTJsonExporter::visit(ContractDefinition const& _node)
 		make_pair("abstract", _node.abstract()),
 		make_pair("baseContracts", toJson(_node.baseContracts())),
 		make_pair("contractDependencies", getContainerIds(_node.annotation().contractDependencies | ranges::views::keys)),
+		// Do not require call graph because the AST is also created for incorrect sources.
+		make_pair("emittedEvents", getContainerIds(_node.interfaceEvents(false))),
```

Comment:
```suggestion
		make_pair("definedAndEmittedEvents", getContainerIds(_node.interfaceEvents(false))),
```
Would this fit better?

Reply:
I guess the best would be to distinguish between `defined` and `emitted` events.

Reply:
> Would this fit better?

It's clearer than `emittedEvents`, but maybe we should go for `usedEvents` for consistency with `usedErrors`?

> I guess the best would be to distinguish between `defined` and `emitted` events.

Perhaps, but we do not distinguish it for errors either so I think it's fine like that for events too.